src/app/shared/services/claims-facade/claims-facade.service.spec.ts

```diff
@@ -50,56 +50,24 @@ describe('ClaimsFacadeService', () => {
         claimType: createClaimType(claimType),
       } as Claim);
     };
-
-    it('should return empty list when getting empty list', async () => {
-      claimsServiceSpy.getUserClaims.and.returnValue(Promise.resolve([]));
-      expect(await service.addStatusIfIsSyncedOffChain([])).toEqual([]);
-    });
-
-    it('should return not changed list when UserClaims is empty', async () => {
```

Comment:
Those tests are removed because they are no longer valid or they are testing nothing in your opinion?

Reply:
Oh I just changed them actually because it’s not a list anymore, it’s mapping over each claim rather than the list. It’s still testing the map though I think…

Reply:
Yeah, right! Thanks for clarify, you can merge it imo :)